torchtitan/val.py

```diff
@@ -0,0 +1,97 @@
+import time
+from torch.nn.functional import cross_entropy
+
+def loss_fn(pred, labels):
```

Comment:
This function should be the same as what is used in train, separately specifying it here can lead to issues if we make changes. Let's pull the train loss_fn and this one into a separate file and import it in both files.